src/Analyzer.TemplateProcessor/PlaceholderInputGenerator.cs

```diff
@@ -186,7 +186,8 @@ internal static InsensitiveDictionary<JToken> GeneratePlaceholderDeploymentMetad
                 { "subscription", new JObject(
                     new JProperty("id", "/subscriptions/00000000-0000-0000-0000-000000000000"),
                     new JProperty("subscriptionId", "00000000-0000-0000-0000-000000000000"),
-                    new JProperty("tenantId", "00000000-0000-0000-0000-000000000000")) },
+                    new JProperty("tenantId", "00000000-0000-0000-0000-000000000000"),
+                    new JProperty("displayName", "aSubscriptionName")) },
                 { "resourceGroup", new JObject(
```

Comment:
There are several other metadata fields supported in templates, but not initialized here. Can we fill out this section completely to get ahead of other parsing failures later?